CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ if(UNIX)
     #get rid of any RPATH definations already
     set_target_properties(hipify-clang PROPERTIES INSTALL_RPATH "")
     #set RPATH for the binary
-    set_target_properties(hipify-clang PROPERTIES LINK_FLAGS "-Wl,--disable-new-dtags -Wl,--rpath,$ORIGIN/../lib" )
+    set_target_properties(hipify-clang PROPERTIES LINK_FLAGS "-Wl,--enable-new-dtags -Wl,--rpath,$ORIGIN/../lib" )
```

Comment:
Should this fix go with 5.5?

Reply:
Since most of the ROCm libraries and binaries do have RUNPATH(which is recommended). Will bring this also to 5.5